### PR TITLE
Master

### PR DESCRIPTION
**Pull request by npstcole**
The site import function for passes locations was failing because the
development environment didn’t have any data on a fresh site install.
The code was written assuming data existed and would fail. The fix
checks the record count before  proceeding.

### CODE REVIEW

**Comment by OpenGlobe** [comment]
@khandelwal @cacraig - here are Trevor's latest changes to the repo.  Colin, I went ahead and added you as an admin to this repo.  Perhaps we can chat further a bit when we meet this afternoon - thanks!

**Comment by OpenGlobe** [comment]
@cacraig @khandelwal

File: cf.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 echo '-----Starting APP ----------'
 cd ekip
-newrelic-admin run-program waitress-serve --port=$VCAP_APP_PORT ekip.config.wsgi:application
+waitress-serve --port=$VCAP_APP_PORT ekip.config.wsgi:application
```
Comment (cacraig) [inline cf.sh:3]: @khandelwal What is your opinion on this? Are we not using newrelic anymore?

**Comment by cacraig** [comment]
See #233 .

**Pull request merged**